docs/examples/.eslintrc

```diff
@@ -31,6 +31,7 @@
     "Label",
     "ListGroup",
     "ListGroupItem",
+    "ListGroupItemHeading",
```

Comment:
Does there?  The `ListGroupItemHeading` made sense because it wraps the content in an `h4` and adds appropriate classes. Part of the problem before was the ListGroupItem's children were receiving a wrapper span that broke block elements.